portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsFieldsets.testcase

```diff
@@ -398,14 +398,83 @@ definition {
 	}
 
 	@description = "Verify that an Allow Multiple Selections Property persists after being added to a Form via Fieldset"
-	@ignore = "Test Stub"
 	@priority = "4"
 	test PreserveTheMultipleSelectionsProperty {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-133473 PreserveTheMultipleSelectionsProperty pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		Form.gotoElementSetsTab();
+
+		FormsAdminNavigator.gotoAddElementSet();
+
+		Form.editName(formName = "Element Set 1");
+
+		Form.addTextField(fieldPositionNumber = "0");
+
+		Form.closeSidebar();
 
+		Form.gotoAddField(
+			fieldPositionNumber = "1",
+			fieldType = "Select from List");
+
+		FormFields.addSingleSelectionOption(
+			optionName = "Option 1",
+			optionValue = "1");
+
+		FormFields.addSingleSelectionOption(
```

Comment:
@fmarcio is it possible to use `FormFields.editOptionsRow` instead?

Reply:
> @fmarcio is it possible to use `FormFields.editOptionsRow` instead?

@carolmariaabb  Fixed!